kustomize/monitoring/base/logger.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: serving.knative.dev/v1
```

Comment:
Based on the Himss demo requirements, we should be able to monitor external dkube deployments where no modelmonitor is deployed.

So, lets move this out of monitoring. Put it under dkube or kubeflow.  Including the namespace in the yaml below.

Reply:
there is also kustomize-nfs. Do we need to have this there as well?